english/forest/scream/run/run.md

```diff
@@ -1,4 +1,16 @@
-Since you know you cannot outrun a tiger. You decide to run toward it 
+Since you know you cannot outrun a tiger. You decide to run toward it
 making as much noise as possible in a way to scare it off!!
 The tiger tilts its head slightly to the left and watches you with that
-curious expression so typical of three year olds. When you reach the tiger it lifts an expectant paw, and....
\ No newline at end of file
+curious expression so typical of three year olds. When you reach the tiger it lifts an expectant paw, and....
+
+You're now flying backwards!
+
+You forgot that you were still attached to the parachute, which sling-shotted you backwards, just as you reached the end of the tether.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.